### ISSUE TITLE
Suggestions/questions re configuring the frontpage template

### ISSUE THREAD

**Issue by fedorov** [Phlow/feeling-responsive#181]
In the process of customizing the theme for our web site, we ran into the following:
- we did not want to see the "More" buttons under each of the front page widgets. We disabled them manually by commenting out this line https://github.com/Phlow/feeling-responsive/blob/gh-pages/_includes/_frontpage-widget.html#L19. Would it make sense to have the widget button optional, and show it only if the `url` attribute is initialized? If this makes sense, and someone can let me know what syntax that file follows, I can make a pull request
- it would be nice if it was possible to configure front page with a paragraph that spans all three columns. Would it make sense to have this as a configurable feature of the front page template?

Thanks again for this wonderful template!

**Comment by Phlow** [comment]
Hello @fedorov – I added your suggested functionality for the widget, which I think is a good idea. But I won't add another banner paragraph – I don't want to blow up the code and confuse people with more functionality. But you should easily change the callforaction-include – it spans across all three/twelve columns.

Does it help you like this?

**Comment by Phlow** [comment]
@fedorov This is the commit by the way ;)

Commit https://github.com/Phlow/feeling-responsive/commit/b26f7e5f95931baa8ab6c1d1bca7f25d2e3bfff3

**Comment by fedorov** [comment]
@Phlow thanks a lot for such a quick response, and for the pointer on how to achieve what we need. I will look at it and let you know if I have further questions!

**Comment by fedorov** [comment]
>  But you should easily change the callforaction-include – it spans across all three/twelve columns.

@Phlow can you give me an example how it can be used? I am not familiar with the "under the hood" content. Is there some guide? I can't easily find it in the web site documentation.

**Comment by Phlow** [comment]
Hey @fedorov — Just edit 
https://github.com/Phlow/feeling-responsive/blob/gh-pages/_layouts/frontpage.html

The theme uses the foundation 5 grid. Edit this here and pur your own Code in that place. Delete the Liquid commands around it. 



~~~
{% comment %}
*
* First check, if there is a call for action-button
*
{% endcomment %}
{% if page.callforaction.url contains 'http' %}
{% assign url = '' %}
{% else %}
{% capture url %}{{ site.url }}{{ site.baseurl }}{% endcapture %}
{% endif %}
{% if page.callforaction %}
    <div class="row t60 b60">
        <div class="small-12 text-center columns">
            <a class="button large radius {{ page.callforaction.style }}" href="{{ url }}{{ page.callforaction.url }}"{% if page.callforaction.url contains 'http' %} target="_blank" {% endif %}>{{ page.callforaction.text }}</a>
        </div><!-- /.small-12.columns -->
    </div><!-- /.row -->
{% endif %}
~~~

**Comment by fedorov** [comment]
@Phlow thank you for the hints ... I have to say though that I have zero experience with "foundation" or "liquid", and I still don't know how to change it :-/ 

The fact that you mention foundation and liquid hints me toward this documentation page to try to understand how to configure this: https://foundation.zurb.com/sites/docs/v/5.5.3/components/grid.html and https://shopify.github.io/liquid/. I am sure I can figure it out, will look into that...

**Comment by fedorov** [comment]
I figured it out, it was indeed very simple, once I spent some time figuring it out. In case anyone else needs it, here is the solution: https://github.com/OSIPI/osipi.github.io/commit/d1a73a3746fcc4ea6ea1b3b5a60dae9dfa48fd29

@Phlow I still think adding this preamble paragraph to the template makes sense, would have saved me some time, and I think it is functionality that is commonly needed. But I understand your perspective as well.
